bin/dartdoc.dart

```diff
@@ -122,8 +129,10 @@ main(List<String> arguments) async {
       : new PackageMeta.fromDir(inputDir);
 
   if (!packageMeta.isValid) {
-    stderr.writeln(
-        ' fatal error: Unable to generate documentation: ${packageMeta.getInvalidReasons().first}.');
+    stderr
+        .writeln(' fatal error: Unable to generate documentation: ${packageMeta
```

Comment:
This is horrible looking now.  I'd rather us not use expansion for multi-line expressions.

Reply:
Yup, this was an auto-dartfmt. I updated to use a local var instead to hold the error text.